ui/src/app/applications/components/utils.tsx

```diff
@@ -45,17 +45,20 @@ export function helpTip(text: string) {
 }
 export async function deleteApplication(appName: string, apis: ContextApis): Promise<boolean> {
     let confirmed = false;
-    const propagationPolicies: {name: string; message: string}[] = [
+    const propagationPolicies: {name: string; id: string; message: string}[] = [
         {
             name: 'Foreground',
+            id: 'foreground',
```

Comment:
Hi @chetan-rns , looks good. Just one last minor suggestion is to make these ids distinct. (I think they are actual 'id' attributes on the `<input>` element.

So for example, these ids can be 'appDeleteForeground', 'appDeleteBackground', 'appDeleteNonCascading'
or something like that, and for the resource delete ids, they can be:
'resourceDeleteForeground', etc.

Reply:
I've updated the foreground id for resources. Thanks!